### PR TITLE
Add current Contributers to the List

### PR DESCRIPTION
**Pull request by fabiankaegy**
I have updated the List of contributors in the repo to reflect the current people who appear as contributors on GitHub

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,4 +2,14 @@
 
 | GitHub Username | WordPress.org Username|
 | --------------- | --------------------- |
-| @example | @example |
+| @carolinan | @poena |
+| @melchoyce | melchoyce |
```
Comment (pattonwebz) [inline CONTRIBUTORS.md]:
```suggestion
| @melchoyce | @melchoyce |
```

**Review by pattonwebz — CHANGES_REQUESTED** [review]
Looks good to me, thanks for doing this :)

I have just one change requiest - I think there is a single `@` missing.

**Comment by fabiankaegy** [comment]
@pattonwebz Great catch  👍 I have committed your suggestion

**Review by pattonwebz — APPROVED** [review]
LGTM :+1: 

:shipit:

**Comment by aristath** [comment]
Do we even want a contributors file? There will be a lot of people contributing here, and not all of them with code...

**Comment by pattonwebz** [comment]
Yeah we will want a contributors file here because this is how we track who gets props at core commit time. People who contribute to issues, and comment etc should also be added too :)

**Comment by aristath** [comment]
Makes sense :+1:

**Pull request merged**